NBitcoin/RPC/RPCClient.cs

```diff
@@ -1415,6 +1420,33 @@ public async Task<GetTxOutResponse> GetTxOutAsync(uint256 txid, int index, bool
 			};
 		}
 
+		/// <summary>
+		/// Returns statistics about the unspent transaction output (UTXO) set
+		/// </summary>
+		/// <returns>Parsed object containing all info</returns>
+		public GetTxOutSetInfoResponse GetTxoutSetInfo()
+		{
+			return GetTxoutSetInfoAsync().Result;

```

Comment:
.GetAwaiter().GetResult() so the exception does not get wrapped inside AggregateException